src/widgets/allergies/allergy-form.component.tsx

```diff
@@ -241,7 +242,11 @@ export function AllergyForm(props: AllergyFormProps) {
           background: "var(--omrs-color-bg-medium-contrast)"
         }}
       >
-        <form onSubmit={handleCreateFormSubmit} ref={formRef}>
+        <form
```

Comment:
same thing here as above - is it necessary for this component to know about the workspace?